pyproject.toml

```diff
@@ -41,11 +41,17 @@ keywords = [
 ]
 dynamic = ["version"]
 dependencies = [
-    "elastic-transport>=9.1.0,<10",
+    # TODO revert before merging/releasing
+    "elastic-transport @ git+https://github.com/pquentin/elastic-transport-python.git@trio-support",
```

Comment:
Adding a comment here just to remind you that need this to be edited.

Reply:
Thanks! There's no alert or linter, no.